Userland/Libraries/LibJS/Tests/time-from-year.js

```diff
@@ -0,0 +1,4 @@
+test("YearFromTime(TimeFromYear(t)) <= t", () => {
+    let d = new Date(-1112, 11, 31);
+    expect(d.toGMTString()).toBe("Fri, 31 Dec -1112 00:14:44 GMT");
```

Comment:
```
ExpectationError: toBe: expected _Fri, 31 Dec -1112 00:14:44 GMT_, got _Fri, 31 Dec -1112 00:00:00 GMT_
```

This returns `Fri, 31 Dec -1112 00:01:15 GMT` here (UTC+1), and yet another result on CI. Might be best to test this another way (`getDate()` should also trigger the crash) instead of introducing a possibly flaky test.

Reply:
do the `UTC` variants use always the same reference? Or should I use `getDate()` and the like?

Reply:
Hmm, I think the issue may be that we're creating a localtime-`Date` - this may work reliably: `new Date(Date.UTC(-1112, 11, 31)).toGMTString() // "Fri, 31 Dec -1112 00:00:00 GMT"

Just saw your force-push, feel free to keep it as-is if that passes CI now :)

Reply:
`Date.UTC` it is then